.gitignore

```diff
@@ -9,4 +9,6 @@ node/
 package-lock.json
 cypress/screenshots
 cypress/videos
-node_modules/
\ No newline at end of file
+node_modules/
+src/test/java/org/openmrs/contrib/qaframework/RunTest.java
+src/test/resources/org/openmrs/uitestframework/
```

Comment:
is there any local config u need to have this usable?

Reply:
> is there any local config u need to have this usable?

@kaweesi not sure what you meant by this. Did you mean enabling them in the future to be among the tracked files.

Reply:
> is there any local config u need to have this usable?

Hey @kaweesi this could be a cache. Tested this merge by pulling it from upstream and merged it in my testing dev environment but fails to meet the target goal. @HerbertYiga do you mind sharing your result on this after testing. Otherwise we may need to revert this merge and add some technical work